docs/vaahflutter/basics/getting-started.md

```diff
@@ -9,22 +9,34 @@
 
 ## How to run the app
 
-### Step 1. Clone repository
+### Step 1. Clone Repository
 
 To clone the repository, you'll need to run the below command in your terminal. Make sure [git](https://git-scm.com/downloads) is installed in your system.
 
 ```bash
 git clone ...
 ```
 
-### Step 2. Run the app
+### Step 2. Basic Configuration
+
+There are two ways developers can integrate VaahFlutter in their project.
+
+1. Clone the repository and build your APP on top of it
```

Comment:
@we-prajapati-c001 change `clone` to `download`

Reply:
change `build` to `develop`

Reply:
yes sir